src/utils/matchingUtils.js

```diff
@@ -0,0 +1,22 @@
+import { getMatchingHistory } from '@/api/matchingApi';
+
+//n승 n무 n패 계산
+export async function getMatchingRecordStats() {
+  const RECORDS = await getMatchingHistory();
+
+  let win = 0,
+    draw = 0,
+    lose = 0;
+
+  for (const record of RECORDS) {
+    if (record.matchingResult === 'WIN') {
+      win++;
+    } else if (record.matchingResult === 'DRAW') {
+      draw++;
+    } else if (record.matchingResult === 'LOSE') {
+      lose++;
+    }
+  }
```

Comment:
(수정 요청은 아닙니다!) 재할당이 가능한 let을 사용해도 좋지만, Vue의 변경이 가능한 ref객체를 사용해도 좋을 것 같습니다!!
+ 데이터가 배열이라는 점을 고려해서 아래와 같이 forEach메소드로 계산을 할 수도 있을 거 같습니다!

```suggestion
  const win = ref(0);
  const draw = ref(0);
  const lose = ref(0);

  RECORDS.forEach(record => {
    const { matchingResult } = record;
    switch (matchingResult) {
      case 'WIN':
        win.value++;
        break;
      case 'DRAW':
        draw.value++;
        break;
      case 'LOSE':
        lose.value++;
        break;
    }
```

Reply:
말씀해주신 방법도 좋을 것 같네요!! 알려주셔서 감사합니당! 저는 단순 계산용 함수라서 ref를 사용할 생각을 못했던 것 같습니다!

Reply:
아 말씀대로 단순 계산용 함수니까 ref를 사용하지 않는게 더 좋을거 같네요! 제가 잘못 생각했습니다ㅎㅎ🙈